c/src/measure/stats/gitstats.cpp

```diff
@@ -5,6 +5,7 @@
 
 #include <git2.h>
 #include <sha1.h>
+#include <zip.h>
```

Comment:
CI reports some issues with linking Zlib

Reply:
Issues with zlib should be fixed now :). It did not link correctly on some platforms.